decidim-core/app/helpers/decidim/searches_helper.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Decidim
+  # A Helper to render and link to resources.
+  module SearchesHelper
+    def radio_checked?(resource_type_radio_value)
```

Comment:
This method doesn't seem to be covered by tests, and I cannot find any other use of it in this PR. Remove it maybe?